<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>321</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Free Hash Checker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon/icon.png</normaloff>:/icon/icon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>21</height>
       </size>
      </property>
      <widget class="QWidget" name="MainTab">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/home/home.png</normaloff>:/home/home.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Home</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_MainTab">
        <item>
         <widget class="QGroupBox" name="groupBoxHashCalculation">
          <property name="title">
           <string>Hash Calculation</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="labelFileExplore">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="resources.qrc">:/folder/opened-folder.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditFileExplore">
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>Select a file to begin</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonSelectFile">
               <property name="text">
                <string>Select file</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/select/import-file.png</normaloff>:/select/import-file.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>21</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="buttonClearHashBox">
               <property name="text">
                <string>Clear</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/clear/eraser.png</normaloff>:/clear/eraser.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>21</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditHashBox">
               <property name="text">
                <string/>
               </property>
               <property name="maxLength">
                <number>128</number>
               </property>
               <property name="readOnly">
                <bool>false</bool>
               </property>
               <property name="placeholderText">
                <string>Hash will be displayed here</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonCopyToClipboard">
               <property name="text">
                <string>Copy</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/copy/copy.png</normaloff>:/copy/copy.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>21</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QComboBox" name="comboBoxHashChoices">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="editable">
                <bool>false</bool>
               </property>
               <property name="currentText">
                <string>MD5</string>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
               </property>
               <property name="iconSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="frame">
                <bool>true</bool>
               </property>
               <item>
                <property name="text">
                 <string>MD5</string>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/rhombus/rhombus.png</normaloff>:/rhombus/rhombus.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>SHA1</string>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/rhombus/rhombus.png</normaloff>:/rhombus/rhombus.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>SHA224</string>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/rhombus/rhombus.png</normaloff>:/rhombus/rhombus.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>SHA256</string>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/rhombus/rhombus.png</normaloff>:/rhombus/rhombus.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>SHA384</string>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/rhombus/rhombus.png</normaloff>:/rhombus/rhombus.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>SHA512</string>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/rhombus/rhombus.png</normaloff>:/rhombus/rhombus.png</iconset>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QProgressBar" name="progressBarHashCaclulation">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="textVisible">
                <bool>true</bool>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="textDirection">
                <enum>QProgressBar::TopToBottom</enum>
               </property>
               <property name="format">
                <string>%p%</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonHashCalculate">
               <property name="text">
                <string>Calculate</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/calculate/drawing-compass.png</normaloff>:/calculate/drawing-compass.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>21</height>
                </size>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="default">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxHashMatcher">
          <property name="title">
           <string>Hash Matcher</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="buttonClearCheckHashBox">
             <property name="text">
              <string>Clear</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/clear/eraser.png</normaloff>:/clear/eraser.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>21</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditCheckHashBox">
             <property name="maxLength">
              <number>128</number>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="placeholderText">
              <string>Paste hash here</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonCheckHash">
             <property name="text">
              <string>Check</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/check/magnifying-glass.png</normaloff>:/check/magnifying-glass.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>21</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="AboutTab">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/about/info.png</normaloff>:/about/info.png</iconset>
       </attribute>
       <attribute name="title">
        <string>About</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="opaqueResize">
           <bool>false</bool>
          </property>
          <property name="childrenCollapsible">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="developerName">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span
                                                                style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Developer:&lt;/span&gt;
                                                                Rizwan Hasan&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                                            </string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="developerEmail">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span
                                                                style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Email:
                                                                &lt;/span&gt;rizwan.hasan486@gmail.com &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                                            </string>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="logoCreditName">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span
                                                                style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Logo
                                                                credit: &lt;/span&gt;Sakib Khan Inan &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                                            </string>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="logoCreditEmail">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span
                                                                style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Email:
                                                                &lt;/span&gt;sakib.khaninan@hotmail.com &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                                            </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="icons8Credit">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span
                                                                style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Icons
                                                                credit: &lt;/span&gt;Icons8&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                                            </string>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget1">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="hashCheckerHeader">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resources.qrc">:/header/header.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>GitHub</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QLabel" name="sourceCodeLink">
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p
                                                                                    align=&quot;center&quot;&gt;&lt;a
                                                                                    href=&quot;https://github.com/Rizwan-Hasan/Free-Hash-Checker&quot;&gt;&lt;img
                                                                                    src=&quot;:/github/github.png&quot;/&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                                                                </string>
                   </property>
                   <property name="openExternalLinks">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_4">
                <property name="title">
                 <string>Version</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QLabel" name="applicationVersion">
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p
                                                                                    align=&quot;center&quot;&gt;&lt;span
                                                                                    style=&quot; font-size:22pt;
                                                                                    font-weight:600;&quot;&gt;0.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                                                                </string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="licenseTextBrowser">
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="minimumSize">
    <size>
     <width>450</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>326</height>
    </size>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonClearCheckHashBox</sender>
   <signal>clicked()</signal>
   <receiver>lineEditCheckHashBox</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
